### DISCUSSION TITLE
ERRrun return error

### DISCUSSION THREAD

**Discussion by mat194** [LAPKB/Pmetrics#228] [General]
![image](https://github.com/LAPKB/Pmetrics/assets/89579560/eee75ebe-4d30-4e9c-afaf-165bb878e877)
Hello everyone! I've been attempting to use the nERRrun() function to estimate error coefficients from my data but encountered an error. Could anyone suggest what might be going wrong? Is there something I might be missing about how it’s supposed to work? Any advice would be greatly appreciated!

**Comment by mnneely** [comment]
ERRrun is one of the oldest functions in the package, left over from some of the original USC*PACK code, and we haven't updated it in a long time, I'm afraid. It will not survive another round. Estimating the coefficients from your data isn't really a good idea. The estimates you get are entirely dependent on the model, so it is completely circular. If you don't have the assay validation data, use the suggested values: C0 = 0.5 * lowest output value in the data or 0.5 LOQ if you know it. C1 = 0.1 or 0.08, which is a 10% or 8% CV. Most assays are at least that, often down around 3%. C2, C3 = 0.
